flake.nix

```diff
@@ -3,7 +3,7 @@
     common.url = "github:nammayatri/common";
 
     # Backend inputs
-    shared-kernel.url = "github:nammayatri/shared-kernel";
+    shared-kernel.url = "github:nammayatri/shared-kernel/bdeeb5869a270b9654fc92882ec086514000051d";
```

Comment:
Only `flake.lock` should be changed. `flake.nix` need not have any changes. You can specify a revision using these instructions:

https://github.com/nammayatri/nammayatri/tree/main/Backend#updating-flake-inputs